src/activities/categorization/Categorization.qml

```diff
@@ -36,6 +36,7 @@ ActivityBase {
 
     property string boardsUrl: "qrc:/gcompris/src/activities/categorization/resource/"
     property bool vert: background.width < background.height
+    property string datasetURL: "../../GCompris-qt/src/activities/categorization/resource/board/"
```

Comment:
I shifted from using opendir() to QDir, but it still doesn't work for the qrc path.
Code - http://pastebin.com/JavNCD4G